### ISSUE TITLE
enabled_methods not behaving as expected

### ISSUE THREAD

**Issue by stymo** [marcgibbons/django-rest-swagger#73]
Hi,

This may simply be my misunderstanding of the intended behavior, but I'm not getting the results I expect from the ``enabled_methods`` setting. I have included only "get" in this list, but my api docs include the "Try it out!" button for both POST and GET methods.  I was hoping the point of this setting would be to allow me to show docs for the POST method, but not allow users to actually call that method from the docs.  BIG disclaimer, I'm using django 1.4, which I know is unsupported, but it looks like a lot of this setting is implemented in javascript, so I don't know if that would matter so much.

Here's from my settings.py:
```
...
SWAGGER_SETTINGS = {
    "enabled_methods": [
        "get",
    ]
}
...
```
And here's what my docs look like:
![screen shot 2014-03-13 at 10 40 11 am](https://f.cloud.github.com/assets/232289/2410470/82f79732-aabd-11e3-95bb-6504f735fbd8.png)

Am I doing anything wrong, besides using the wrong django version?  Or is there an issue here?

**Comment by ankitml** [comment]
Sigh!! The issue persists even today.

**Comment by edmorley** [comment]
Ok so there are two issues:

1) The django-rest-swagger setting called `enabled_methods` actually sets the swagger-ui's `supportedSubmitMethods` option. This option isn't about controlling which methods are displayed in Swagger (ie so this is unrelated to #142), but instead which methods have a "try it out" button. I think we should rename the setting (and clarify [the docs here](https://django-rest-swagger.readthedocs.io/en/latest/settings.html#enabled-methods)) to more accurately describe the intended effect.

2) Even though defining `enabled_methods` correctly causes `supportedSubmitMethods` to be set in the inline JS (as can be confirmed by viewing the source on a rendered django-rest-swagger page), this variable isn't actually used properly in the old version of swagger-ui used by django-rest-swagger:
https://github.com/marcgibbons/django-rest-swagger/search?utf8=%E2%9C%93&q=supportedSubmitMethods
...compare this to more recent swagger-ui:
https://github.com/swagger-api/swagger-ui/blob/52f3b6d50ec727a8f4d8ddd06388a7edb5d4f97e/src/main/javascript/view/OperationView.js#L89

It looks like this regressed in v2.0.0 and was fixed in v2.1.0:
https://github.com/swagger-api/swagger-ui/pull/349

#309 has been filed for updating to a newer version of swagger-ui, which would fix this issue.
